libs/sdk-ui-ext/src/internal/components/configurationControls/comparison/calculation/CalculationControl.tsx

```diff
@@ -0,0 +1,54 @@
+// (C) 2023 GoodData Corporation
+import React, { useMemo } from "react";
+import { useIntl } from "react-intl";
+
+import { CalculationType } from "@gooddata/sdk-ui-charts";
+
+import { messages } from "../../../../../locales.js";
+import { calculationDropdownItems } from "../../../../constants/dropdowns.js";
+import DropdownControl from "../../DropdownControl.js";
+import CalculationListItem from "./CalculationListItem.js";
+
+import { COMPARISON_CALCULATION_TYPE_VALUE_PATH } from "../ComparisonValuePath.js";
+import { IComparisonControlProperties } from "../../../../interfaces/ControlProperties.js";
+import { IVisualizationProperties } from "../../../../interfaces/Visualization.js";
+
+interface ICalculationControlProps {
+    disabled: boolean;
+    properties: IVisualizationProperties<IComparisonControlProperties>;
+    pushData: (data: any) => any;
```

Comment:
I saw that we started to use `(data: IPushData) => void;`. Possibly, you can define `export type IPushDataCallback = (data: IPushData) => void;` at the place where `IPushData` is defined and use it here and in future cases.

Reply:
I see that you have other places where you use it without `any` :) Please, align all of them. Thank you

Reply:
Fixed as per your suggestion!
Thank you so much!